embedding/embedlite/utils/BrowserChildHelper.cpp

```diff
@@ -261,6 +262,9 @@ BrowserChildHelper::InitBrowserChildHelperMessageManager()
   }
   root->SetParentTarget(scope);
 
+  RefPtr<JSActorService> wasvc = JSActorService::GetSingleton();
+  wasvc->RegisterChromeEventTarget(scope);
```

Comment:
Did you check how it goes in nsFrameLoader::EnsureMessageManager? Does that InProcessBrowserChildMessageManager::Create get called ?

Reply:
That was the eventual source of the fix.